### PR TITLE
test(integration): Add coverage instrumentation to Docker images

### PR DESCRIPTION
**Pull request by alfrunes**
Uses the test images for running `test-integration`

### CODE REVIEW

File: backend/Makefile [hunk 1]
```diff
@@ -66,6 +66,7 @@ $(TEST_ACCEPTANCE_TARGETS):
 	@$(MAKE) -C services/$(subst -test-acceptance,,$@) test-acceptance-run
 
 .PHONY: test-integration
+test-integration: MENDER_IMAGE_TAG=$(MENDER_IMAGE_TAG_TEST)
```
Comment (merlin-northern) [inline backend/Makefile:69]: do we need export here?

File: .gitlab-ci.yml [hunk 2]
```diff
@@ -175,7 +175,7 @@ test:backend:integration:
   before_script:
     - apk add make bash git curl
     - *dind-login
-    - make -C backend -j 4 docker-pull
+    - make -C backend -j 4 docker-pull MENDER_IMAGE_TAG=$MENDER_IMAGE_TAG_TEST
```
Comment (merlin-northern) [inline .gitlab-ci.yml:178]: oh nice, it took me hours and you did it in 5s thank you; you are my hero.

**Review by merlin-northern — APPROVED** [review]

File: backend/Makefile (hunk 1 above)
Comment (alfrunes) [reply backend/Makefile:69]: No, the target is not forking another make process.

File: .gitlab-ci.yml (hunk 2 above)
Comment (alfrunes) [reply .gitlab-ci.yml:178]: Sorry, I have on my todo list to write documentation for the new make(1) interface and how the new CI works, but I was trying to get everything in place first.

File: backend/Makefile (hunk 1 above)
Comment (alfrunes) [reply backend/Makefile:69]: Sorry, you're totally right! I didn't think of the variable not being exposed to the script this way.

**Pull request merged**